runners/apex/src/main/java/org/apache/beam/runners/apex/ApexPipelineOptions.java

```diff
@@ -32,12 +32,6 @@
 
   String getApplicationName();
 
-  @Description("set parallelism for Apex runner")
-  void setParallelism(int parallelism);
```

Comment:
This might make an interesting discussion, depending on what `parallelism` means here. Each runner handles this differently, and the Beam model quite clearly leaves it undefined so runners can do their own thing. I don't think `PipelineOptions` can (or should) take care of it with that level of detail.

Since it is BEAM-838, no need to solve it here.